lib/window/react.tsx

```diff
@@ -155,14 +155,11 @@ export type WindowProps = PropsWithChildren<{
   defaultHeight?: number;
 }>;
 
-export function Window({
-  window,
-  className,
-  defaultWidth,
-  defaultHeight,
-  children,
-}: WindowProps): ReactNode {
-  const [element, ref] = useNullableState<HTMLElement>();
+export const Window = forwardRef<HTMLDivElement, WindowProps>(function Window(
+  { window, className, defaultWidth, defaultHeight, children },
+  ref,
+): ReactNode {
+  const [element, setElement] = useNullableState<HTMLElement>();
```

Comment:
Please use https://react.dev/reference/react/useImperativeHandle for this instead.